config/environments/development.rb

```diff
@@ -0,0 +1,70 @@
+require "active_support/core_ext/integer/time"
```

Comment:
Read about all the options used in this file

Reply:
ok sir